src/dev_guides/test.md

```diff
@@ -17,11 +17,24 @@
 
 # How to test Parsec
 
-The `tests/all.sh` script executes all tests. [`rustfmt`](https://github.com/rust-lang/rustfmt)
-and [`clippy`](https://github.com/rust-lang/rust-clippy) are needed.
+Parsec relies on a mix of unit, integration and fuzz tests. Unit tests are usually found in the same module as the code they verify. Integration tests can be found in the `tests` directory (along with the code framework required for running them), and come in two flavours: single-provider and all-providers.
```

Comment:
Should we add static tests (`clippy`) and stress tests?

Reply:
`Clippy` and `fmt` are mentioned below, but I did forget about stress tests indeed!